code/modules/fish/fish_items.dm

```diff
@@ -152,15 +152,15 @@
 	name = "shark teeth"
 	desc = "A number of teeth, supposedly from a shark."
 	icon = 'icons/obj/fish_items.dmi'
-	icon_state = "teeth"
```

Comment:
Could you please keep this line? Mapping these in otherwise will make them look weird.